AirCasting/SessionViews/MultiSliderView.swift

```diff
@@ -56,6 +63,11 @@ struct MultiSliderView: View {
         .frame(height: 5)
     }
     
+    func xFor(value: Float, geometry: GeometryProxy) -> CGFloat {
```

Comment:
not sure if a good name. what is value? what exactly is `x`

Reply:
I chaged it to `    func calculateXAxisSize(thresholdValue: Float, geometry: GeometryProxy) -> CGFloat {...}`
I hope it's good enough :)